### ISSUE TITLE
Users > Pending Approvals: User Roles here?

### ISSUE THREAD

**Issue by kdegarmo** [GlobalUniversity/ArkTrainingNetwork#99]
Is the user_search_role_list_item going to be used in the pending approvals area or is this just left over from users list item? I'd like to center align all of the fields but if we'll add the role here somehow then I'd re-think that approach.

![image](https://user-images.githubusercontent.com/129225548/228996298-33755479-26d4-45b0-bb20-dd5147bafbe5.png)

**Comment by josh-havens** [comment]
We could add it, but we hadn't planned on it. It might be a good idea though.

Since we can attach a role to an account creation form, it could be convenient to see the role requested show up here based on the role attached to the form they filled out. What do you think?

**Comment by kdegarmo** [comment]
Agreed, I think it could be convenient to see the requested role on first glance. Somewhat related question: should this be somewhat mobile-friendly? That would be a hesitancy I have about adding the role tags for now. I could do a couple of additional things here, I think, to make this even slightly more mobile-friendly in its current state.
